automatic_mode_node:
  ros__parameters:
    targeted_distance: 1.0
    brightness_threshold: 50

battery_node:
  ros__parameters:
    driver_adress: 0x48
    rate: 1
    in_min: 14460
    in_max: 20195
    out_min: 0
    out_max: 100

camera_node:
  ros__parameters:
    camera_index: 0
    framerate: 30
    frame_width: 640
    frame_height: 480

lights_node:
  ros__parameters:
    bcm_camera_light_pin: 4
    bcm_led_light_pin: 5

motors_node:
  ros__parameters:
    driver_adress: 0x40
    a_pwm_channel: 0
    a_in1_channel: 1
    a_in2_channel: 2
    b_pwm_channel: 5
    b_in1_channel: 3
    b_in2_channel: 4

screen_node:
  ros__parameters:
    dc_pin: 24
    rst_pin: 25
    cs_pin: -1
    blk_pin: -1
    baudrate: -1

servomotors_node:
  ros__parameters:
    driver_adress: 0x41
    head_rotation_channel: 0
    neck_top_channel: 1
    neck_bot_channel: 2
    left_eye_channel: 3
    right_eye_channel: 4
    left_arm_channel: 5
    right_arm_channel: 3
    head_rotation_init_angle: 90
    neck_top_init_angle: 0
    neck_bot_init_angle: 30
    left_eye_init_angle: 70
    right_eye_init_angle: 110
    left_arm_init_angle: 140
    right_arm_init_angle: 40

soundbox_node:
  ros__parameters:
    default_volume: 0.2

ultrasonic_sensors_node:
  ros__parameters:
    rate: 20
    bcm_left_trig_pin: 17
    bcm_left_echo_pin: 18
    bcm_right_trig_pin: 22
    bcm_right_echo_pin: 23
    bcm_front_trig_pin: 6
    bcm_front_echo_pin: 12

web_server_node:
  ros__parameters:
    port: 8080
